src/usr/share/batwatch/batwatch-daemon.py

```diff
@@ -127,11 +120,14 @@ def read_configuration_and_create_logger(program_uid, program_gid):
     os.seteuid(os.getuid())
     os.setegid(os.getgid())
```

Comment:
Regain root permissions just before creating the /run directory instead of immediately following creating the log directory.

Reply:
It looks like it already regains root permissions where it should. These two lines appear to be redundant.